content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2873,10 +2874,24 @@ The following example demonstrates manually adding `@ignore` to exclude the `ema
 model User {
   id    Int    @id
   name  String
-  email String @ignore // this field will be excluded
+  email String? @ignore // this field will be excluded
 }
 ```
 
+When adding `@ignore` to a required field, it becomes impossible to add new records to the database, so the `create` and `createMany` methods are removed from the Client API:
```

Comment:
Can we reword this to make the "cause and effect" more obvious? Im not sure if my interpretation is correct; is it something like:
"Adding `@ignore` to any required field makes it impossible to add new records to the database (because the lack of any required field in a record disallows the adding of the record), and thus the `create` and `createMany` methods cannot be used, as they are removed from the Client API:"??

Reply:
That is correct, although the order of things is a bit unexpected in your wording @tanberry.
I made a suggestion to explain that in the initial description of `@ignore` here that should be clearer: https://github.com/prisma/docs/pull/3792